libs/core/src/lib/wizard/wizard.component.ts

```diff
@@ -412,15 +428,27 @@ export class WizardComponent implements AfterViewInit, OnDestroy {
                         '.' + WIZARD_NAVIGATION_CLASS
                     ).clientHeight;
 
-                    scrollTop(
-                        this.wrapperContainer.nativeElement.children[0],
-                        child.offsetTop - wizardNavigationHeight
-                    );
+                    scrollTop(this.wrapperContainer.nativeElement, child.offsetTop - wizardNavigationHeight);
+
+                    if (index === 0) {
+                        this._focusFirstTabbableElement(index);
+                    } else {
+                        // setTimeout needed to allow for smooth scroll animation
+                        setTimeout(() => {
+                            this._focusFirstTabbableElement(index);
+                        }, 500);
+                    }
                 }
             });
             this._setUpScrollListener();
         }
-        this.scrollbar.scroll({ top: 0, behavior: 'smooth' });
+    }
+
+    /** @hidden */
+    private _focusFirstTabbableElement(index = 0): void {
+        const contentContainer = this._elRef.nativeElement.querySelectorAll('.fd-wizard__content')[index];
+        const elToFocus = contentContainer?.querySelector('button, input, select, [tabindex]') as HTMLElement;
```

Comment:
https://github.com/SAP/fundamental-ngx/blob/fefac1929380f482a9786b09513779548281256c/libs/cdk/src/lib/utils/services/tabbable-element.service.ts#L14
You need something like this here?